e2e_tests/tests/cluster/test_exp_continue.py

```diff
@@ -1,176 +1,66 @@
-import tempfile
-from typing import List, Tuple
+from typing import Any, Dict, List, Tuple
 
 import pytest
 
-from determined.common import util
 from determined.common.api import bindings
-from tests import api_utils
-from tests import config as conf
-from tests import detproc
+from determined.experimental import client
+from tests import api_utils, detproc
 from tests import experiment as exp
-
-
-@pytest.mark.e2e_cpu
-def test_continue_config_file_cli() -> None:
```

Comment:
Nit: extra `_args` in the PR description.

> cluster/test_exp_continue:test_continue_config_file_and_args_cli
> converted to a unit test

Reply:
ah yes, will fix when I land it.  Great catch.